apps/microtvm/reference-vm/arduino/README.md

```diff
@@ -24,7 +24,8 @@ microTVM platforms that are supported by [Arduino](https://www.arduino.cc/).
 Arduino VM is published under [tlcpack](https://app.vagrantup.com/tlcpack).
 Here is a list of different release versions and their tools.
 
-(none currently)
+## Versioning
+We use semantic versioning as it is recommended by [Vagrant](https://www.vagrantup.com/docs/boxes/versioning).
```

Comment:
could we state more about when we increment the major revision and when we don't? maybe just make it explicit that changing any SDK component is grounds for a major version rev, and we can add them or make non-breaking changes with minor?